psi4/src/psi4/libmints/angularmomentum.h

```diff
@@ -52,20 +52,16 @@ class AngularMomentumInt : public OneBodyAOInt {
     //! Obara and Saika recursion object to be used.
```

Comment:
They're used to compute the magnetic dipole transition moments, which are needed for rotatory strengths.  [Here's a reference](dx.doi.org/10.1002/wcms.55) in case you're curious.